.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
   - id: black
 
 - repo: https://github.com/pycqa/isort
```

Comment:
Run ``pre-commit autoupdate``. I'm guessing some of them are not up-to-date still 😄

Reply:
Thank you for the tip!